lib/ffmprb/process.rb

```diff
@@ -58,6 +59,17 @@ def duck_audio(av_main_i, a_overlay_i, silence, av_main_o,
             roll in_main
 
             ducked_overlay_volume = {0.0 => volume_lo}
+
+            silence.each_cons(2) do |(silent, next_silent)|
+              silent_part = silent.overlaps || silent
+
+              if silent_part.end_at.nil? or (next_silent.start_at - silent_part.end_at) < audible_sound_min
```

Comment:
how can `silent_part.end_at.nil?` happen? if this is a consideration for ffmpeg data inconsistency, it should be moved upstream.